### ISSUE TITLE
some question

### ISSUE THREAD

**Issue by blockhunts** [zia207/Sentinel--2-Data-Processing#1]
Hi, nice work you have here.

Can i ask you some question by email?
I have some error and maybe it's because my data is not similar with your training data

**Comment by prativadas** [comment]
can you explain how you created the training data from the Sentinel satellite image?

**Comment by zia207** [comment]
Training data was created in R. Below is link. 
https://zia207.github.io/geospatial-data-science.github.io/ground-truth-data-processing.html

**Comment by prativadas** [comment]
Sir, Thank you very much for replying to my query. Can you please clarify
if you created the shapefile (ground truth label) and if yes how did you
create them?



Regards,
Prativa Das
